lib/ecwid_api/client.rb

```diff
@@ -13,7 +13,7 @@ class Client
     extend Forwardable
 
     # The default base URL for the Ecwid API
-    DEFAULT_URL = "https://app.ecwid.com/"
+    DEFAULT_URL = "https://app.ecwid.com"
```

Comment:
The trailing slash in the `DEFAULT_URL` is causing a double slash to be in the url, which `faraday` does not like.